classes/options-general.php

```diff
@@ -591,31 +591,31 @@ class='howto'><?php _e( 'Select "No Limit" to display all results within the sea
 									<!-- Optional Features -->
 									<div class="postbox">
 
-										<h3><?php _e( 'Optional / Experimental Features', 'simplemap' ); ?></h3>
+										<h3><?php _e( 'Optional / Experimental Features', 'SimpleMap' ); ?></h3>
 
 										<div class="inside">
-											<p class="sub"><?php printf( __( 'See %s the Help page%s for an explanation of these options.', 'simplemap' ), '<a href="' . get_bloginfo( 'wpurl' ) . '/wp-admin/admin.php?page=simplemap-help">', '</a>&nbsp;' ); ?></p>
+											<p class="sub"><?php printf( __( 'See %s the Help page%s for an explanation of these options.', 'SimpleMap' ), '<a href="' . get_bloginfo( 'wpurl' ) . '/wp-admin/admin.php?page=simplemap-help">', '</a>&nbsp;' ); ?></p>
 
 											<div class="table">
 												<table class="form-table">
 
 													<tr valign="top">
-														<td width="150"><?php _e( 'Permalinks', 'simplemap' ); ?></td>
+														<td width="150"><?php _e( 'Permalinks', 'SimpleMap' ); ?></td>
 														<td>
 															<label for="enable_permalinks"><input type="checkbox"
 															                                      name="enable_permalinks"
-															                                      id="enable_permalinks" <?php checked( $enable_permalinks ); ?> /> <?php _e( 'Enable location permalinks?', 'simplemap' ); ?>
+															                                      id="enable_permalinks" <?php checked( $enable_permalinks ); ?> /> <?php _e( 'Enable location permalinks?', 'SimpleMap' ); ?>
 															</label>
 															<br/><label for="permalink_slug">
-																<small><?php _e( 'Location permalink folder?', 'simplemap' ); ?></small>
+																<small><?php _e( 'Location permalink folder?', 'SimpleMap' ); ?></small>
 																<input type="text" name="permalink_slug"
 																       id="permalink_slug"
 																       value="<?php echo esc_attr( $permalink_slug ); ?>"/></label>
 														</td>
 													</tr>
 
 													<tr valign="top">
-														<td width="150"><?php _e( 'Location Taxonomies', 'simplemap' ); ?></td>
+														<td width="150"><?php _e( 'Location Taxonomies', 'SimpleMap' ); ?></td>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 127 characters

<!-- Comment by [SideCI](https://sideci.com) -->